includes/EDD_SL_Plugin_Updater.php

```diff
@@ -307,6 +314,8 @@ private function api_request( $_action, $_data ) {
             $request = false;
         }
 
+	set_transient( $cache_key, $request, 60*60*24 );
```

Comment:
On second thought, talked to @pippinsplugins about this and I think we're prefer to match Core's cache and make it `HOUR_IN_SECONDS * 12`.